### PR TITLE
David fix

### PR DESCRIPTION
**Pull request by nw0428**
Please review

[Review on Reviewable](https://reviewable.io/reviews/nw0428/sibspring2016/1)

### CODE REVIEW

File: david.html [hunk 1]
```diff
@@ -58,8 +58,8 @@
           </div>
           <div class="span6 center">
             <center>
-              <img src="ie_logo.gif">
-              <img src="ns_logo.gif">
+              <img src="img/david/ie_logo.gif">
+              <img src="img/david/ns_logo.gif">
```
Comment (nw0428) [inline david.html:62]: This image sucks.

**Comment by nw0428** [comment]
LGTM

**Comment by rhymiz** [comment]
This sucks. Fix it.

**Comment by obadiahkopchak** [comment]
looks good to me

**Comment by jdorancy** [comment]
Looks good!

**Comment by ZPendragon** [comment]
LGTM

**Comment by JQuezada0** [comment]
Looks good to me

**Comment by gbattra** [comment]
< wise ass remark >

**Comment by spencekarl** [comment]
LGTM

**Comment by twiz5000** [comment]
looks good to me

**Comment by saturne** [comment]
Good to me

**Pull request merged**